tools/backup/templates/backup.yaml

```diff
@@ -57,17 +60,21 @@ spec:
                   value: "{{ .Values.checkLabelScanStore }}"
                 - name: CHECK_PROPERTY_OWNERS
                   value: "{{ .Values.checkPropertyOwners }}"
+              {{- if .Values.secretName }}
               volumeMounts:
```

Comment:
There is no way to mount a volume in the backup utility, if a user is trying to backup a big database and the local storage does not have enough space then the backup job will fail. We are currently facing this problem and so solve it we mounted a volume to the backup pod.

Reply:
This is the events I can see in the backup pod: 
  Warning  Evicted    7m20s  kubelet            The node had condition: [DiskPressure].

Reply:
@Mrunal-Yadav that's a good point. I have created a separate issue for that and I will look at it as soon as possible:

https://github.com/neo4j-contrib/neo4j-helm/issues/173